saleor/graphql/product/bulk_mutations/products.py

```diff
@@ -102,6 +103,13 @@ def perform_mutation(cls, _root, info, ids, **data):
 
         return response
 
+    @classmethod
+    def bulk_action(cls, queryset):
+        manager = get_plugins_manager()
```

Comment:
We should get the manager from context.

Reply:
I see mismatches here. For the bulk method you call `product_deleted` for products that will be deleted. But for default product delete mutation you call `product_deleted` after the product was deleted.